main.tf

```diff
@@ -141,25 +148,23 @@ resource "aws_cloudfront_distribution" "fullstory_relay" {
 
   ordered_cache_behavior {
     path_pattern               = "/rec/bundle"
-    allowed_methods            = ["GET", "HEAD", "OPTIONS"]
-    cached_methods             = ["GET", "HEAD", "OPTIONS"]
+    allowed_methods            = ["GET", "HEAD", "OPTIONS", "PUT", "POST", "PATCH", "DELETE"]
```

Comment:
The other distribution changes should be superfluous, but this one is required since POST is needed for the `/rec/bundle` endpoint.